Formula/profanity.rb

```diff
@@ -19,6 +19,8 @@ class Profanity < Formula
     depends_on "libtool" => :build
   end
 
+  option "with-python-plugin-support", "Build with support for python plugins"
```

Comment:
How about just `depends_on :python => :optional`, which will create option text automatically and is less buggy than `depends_on ... if`.